web/web/emails/send_emails.py

```diff
@@ -1,37 +1,25 @@
-import smtplib
-from dotenv import load_dotenv
-load_dotenv()
-
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail, Personalization, Email
 import os
-from email.mime.text import MIMEText
-
-EMAIL_ADDRESS = os.getenv('EMAIL_ADDRESS')
-PASSWORD = os.getenv('PASSWORD')
 
-server = smtplib.SMTP('smtp.gmail.com:587')
-server.ehlo()
-server.starttls()
+def send_email(alert_type, notification_message, email_address):
+    '''sendgrid email setup'''
 
+    message = Mail(
+        from_email=os.getenv('EMAIL_ADDRESS'),
+        to_emails=email_address
+        )
 
-def send_email(subject, message, bcc):
-    '''Simple SMTP set up with gmail'''
+    message.dynamic_template_data = {
+        "subject": "TESS Notification: " + alert_type + " alert",
+        "notification_message": notification_message
+    }
 
-    # Simple set up for basic working email functionality
-    # TODO: styling email, switch to implementing OAuth/Gmail API
+    message.template_id = os.environ.get('SENDGRID_NOTIFICATION_TEMPLATE_ID')
 
     try:
-        server = smtplib.SMTP('smtp.gmail.com:587')
-
-        msg = MIMEText(message)
-        msg['From'] = EMAIL_ADDRESS
-        msg['Subject'] = subject
-
-        server.ehlo()
-        server.starttls()
-        server.login(EMAIL_ADDRESS, PASSWORD)
-
-        server.sendmail(EMAIL_ADDRESS, bcc, msg.as_string())
-        server.quit()
+        sendgrid_client = SendGridAPIClient(os.environ.get('SENDGRID_API_KEY'))
+        sendgrid_client.send(message)
 
-    except:
-        print('email failed to send')
+    except Exception as e:
```

Comment:
Don't catch a generic exception, be explicit about the exact error you are handling.

Reply:
Fixed, let me know if it looks ok! Thanks